test/testdata/generate.sh

```diff
@@ -118,6 +118,11 @@ function scaffold_test_project {
       $kb edit --plugins=grafana.kubebuilder.io/v1-alpha
   fi
 
+  if [[ $project == "project-v3" || $project == "project-v4" || $project =~ multigroup ]]; then
```

Comment:
WDYT about: 
change this sample `project-v3-addon-and-grafana` name for project-v3-declarative-v1
then, we have a new one as project-v3-metrics were we will add grafana and this one.

Same for v4.

Reply:
Yeah, I think maybe we can deal with this in a separate PR? Since this one maybe hold.

Reply:
@camilamacedo86 Here is the PR to refact `testdata`: https://github.com/kubernetes-sigs/kubebuilder/pull/3086